modules/apps/frontend-token/frontend-token-definition-api/src/main/resources/com/liferay/frontend/token/definition/token-definition.schema.json

```diff
@@ -0,0 +1,139 @@
+{
```

Comment:
Should we also store the `token-values.schema.json` in here?  I would say probably not... but not sure... @Ruben-Pulido? What would be better for you to consume it?

Reply:
I don't think `token-values.schema.json` will go at the infra level, because it's more an application thing. But if we introduce some methods/services to deal with token values, then the answer will be yes.